package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "devicon",
-  "version": "2.15.0",
+  "version": "2.15.1",
```

Comment:
You should reverse all the changes in `package-lock.json` since they are not relevant for icon requests